pandas/io/excel.py

```diff
@@ -130,6 +131,16 @@ def read_excel(io, sheetname=0, header=0, skiprows=None, skip_footer=0,
         either be integers or column labels, values are functions that take one
         input argument, the Excel cell content, and return the transformed
         content.
+    true_values : list, default None
```

Comment:
I see the issue. we say ``list`` here, but in the csv parser we actually accept a scalar as well. So can you update this doc-string (and read-csv & in io.rst) for that.